### DISCUSSION TITLE
Android 14, Dexcom g6: Companion app not working

### DISCUSSION THREAD

**Discussion by Agio56** [NightscoutFoundation/xDrip#3155] [Q&A]
Let me start by saying that I never used this mode before updating my Pixel 6 pro to Android 14, so I don't know if the problem I'm experiencing is specific to the latest version of Android.
I use the original Dexcom g6 app and I have activated the display of blood glucose values in notifications as you can see from the screenshot.
![Screenshot](https://github.com/NightscoutFoundation/xDrip/assets/137171485/28e18115-c776-4bcf-b6fe-f6ea4e9d3694)

**Comment by Navid200** [comment]
Go to Android Settings &#8722;> Apps &#8722;> xDrip+ 

Do you see a 3-dot menu at the top right corner of the screen.  If yes, you need to use it to give permission to xDrip and approve.

**Reply by Agio56** [reply]
Already done.

**Reply by Navid200** [reply]
Which version of xDrip are you using?

**Reply by Agio56** [reply]
xDrip-plus-20231024-e34ebac.apk

**Reply by Navid200** [reply]
Can you show a screenshot of the logs please?

**Reply by Agio56** [reply]
![Screenshot_20231026-211328](https://github.com/NightscoutFoundation/xDrip/assets/137171485/376e3603-f72c-4abf-897b-703a18aafbb7)
In the meantime I had switched back to follow share mode.

**Comment by jamorham** [comment]
Could you switch to companion mode, leave it 20 minutes and then upload the logs using the xDrip log upload feature. Thanks.

**Reply by Agio56** [reply]
I just uploaded the log. Thank you
